### PR TITLE
[FSDE-889]Link component

### PR DESCRIPTION
**Pull request by riya-abraham01**
@janibolkvadze Please review

### CODE REVIEW

File: src/components/Link/Link.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+import * as PropTypes from 'prop-types';
+import * as React from 'react';
+
+export default class Link extends React.Component {
+  render() {
+    let download = '';
```
Comment (janibolkvadze) [inline src/components/Link/Link.js:8]: Let's remove the download scenario

File: src/components/Link/Link.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        <span>{this.props.title ? this.props.title : ''}</span>
+      </chi-link>
+    );
+  }
+}
+
+Link.propTypes = {
+  disabled: PropTypes.bool,
+  cta: PropTypes.bool,
+  target: PropTypes.oneOf(['_self', '_blank', '_parent', '_top', 'framename']),
```
Comment (janibolkvadze) [inline src/components/Link/Link.js:43]: I think prototypes are not going to need target prop. Could we add a `click`interaction instead?

`click: PropTypes.func,`

With this the designers will be able to set whatever the interaction is necessary for their prototype cases

File: src/components/Link/Link.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  clickHelp: PropTypes.func,
+  mouseoverHelp: PropTypes.func,
+  mouseleaveHelp: PropTypes.func,
+  alternativeText: PropTypes.string,
+};
+
+Link.defaultProps = {
+  disabled: false,
+  cta: false,
+  download: false,
```
Comment (janibolkvadze) [inline src/components/Link/Link.js:60]:
```suggestion
```

File: src/components/Link/Link.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  helpIcon: false,
+  title: 'Link',
+  size: 'md',
+  target: '_self',
```
Comment (janibolkvadze) [inline src/components/Link/Link.js:64]:
```suggestion
```

File: src/components/Link/presets/0-default.jsx [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+import React from 'react';
+import Link from '../Link';
+
+export default (
+  <Link
+    uxpId="link"
+    disabled={false}
+    cta={false}
+    helpIcon={false}
+    download={false}
```
Comment (janibolkvadze) [inline src/components/Link/presets/0-default.jsx:11]:
```suggestion
```

File: src/components/Link/presets/0-default.jsx [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+    title="Link"
+    size="md"
+    target="_self"
```
Comment (janibolkvadze) [inline src/components/Link/presets/0-default.jsx:14]:
```suggestion
```

File: uxpin.config.js [hunk 7]
```diff
@@ -36,6 +36,7 @@ module.exports = {
         include: [
           'src/components/Alert/Alert.js',
           'src/components/Page-Title/Page-Title.js',
+          'src/components/Link/Link.js',
```
Comment (janibolkvadze) [inline uxpin.config.js:39]: Let's move this down to keep the proper alphabetical order, please

**Review by janibolkvadze — CHANGES_REQUESTED** [review]

File: src/components/Link/Link.js (hunk 1 above)
Comment (riya-abraham01) [reply src/components/Link/Link.js:8]: removed

File: uxpin.config.js (hunk 7 above)
Comment (riya-abraham01) [reply uxpin.config.js:39]: changes added

File: src/components/Link/Link.js [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+    const help = this.props.helpIcon
+      ? (
+        <chi-icon
+          icon="circle-question-outline"
+          onClick={this.props.clickHelp}
+          onMouseEnter={this.props.mouseoverHelp}
+          onMouseLeave={this.props.mouseleaveHelp}>
+        </chi-icon>
+      )
+      : <chi-icon></chi-icon>;
```
Comment (janibolkvadze) [inline src/components/Link/Link.js:17]: Let's import the native react <Icon /> component and reuse it here to solve the empty space issue.
Here is an example

https://user-images.githubusercontent.com/25136131/124928190-b5855200-dfff-11eb-855a-7f811738aad0.mov

**Review by janibolkvadze — CHANGES_REQUESTED** [review]

File: src/constants/classes.js [hunk 9]
```diff
@@ -104,6 +104,9 @@ export const UTILITY_CLASSES = {
         CENTER: '-justify-content-md--center',
         END: '-justify-content-md--end'
     },
```
Comment (janibolkvadze) [inline src/constants/classes.js:106]:
```suggestion
    },
  ALIGN_ITEMS: {
    CENTER: '-align-items--center',
  },
```

**Review by janibolkvadze — CHANGES_REQUESTED** [review]

**Review by janibolkvadze — APPROVED** [review]

**Pull request merged**